### PR TITLE
[Toggl Track] Prepare extensions for new commands

### PR DESCRIPTION
**Pull request by bkeys818**
## Description

This extension was created with a single standalone command. I'm going to be creating PRs that introduce new commands, but first there are some changes that should be made so this command so it can better stand alongside new ones.

- Rename main command to "Start/Stop Time Entry."
- Remove projects for initial list view.
  - Reduces clutter and removes duplicate functionality.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by franzwilhelm** [comment]
@bkeys818 should we do multiple commands, or try to expand the current command?

**Comment by bkeys818** [comment]
@franzwilhelm, I'm thinking multiple commands. I've been planning everything [here](https://github.com/users/bkeys818/projects/7/views/2).

**Comment by pernielsentikaer** [comment]
- Could we keep the `name` of the command `index`, this way we keep ranking and users shortcut

- Doesn't it make more sense to keep the current name?

**Comment by bkeys818** [comment]
@pernielsentikaer, I guess it would hurt to keep the `name` as `index`, but the `title` field definitely needs to be changed.

**Comment by pernielsentikaer** [comment]
That's fine, title can change 🙂

**Comment by pernielsentikaer** [comment]
Is this ready for another review then, or not yet?

**Comment by bkeys818** [comment]
@pernielsentikaer, Yup! Ready to go.

File: extensions/toggl-track/src/components/CreateTimeEntryForm.tsx [hunk 1]
```diff
@@ -29,9 +29,9 @@ function CreateTimeEntryForm({ project, description }: { project?: Project; desc
         billable,
       });
       await showToast(Toast.Style.Animated, "Starting time entry...");
-      revalidateRunningTimeEntry();
       await showToast(Toast.Style.Success, "Started time entry");
```
Comment (pernielsentikaer) [inline extensions/toggl-track/src/components/CreateTimeEntryForm.tsx:32]: Which one of them should we keep? Could you remove one of them 🙂

**Comment by pernielsentikaer** [comment]
@bkeys818 are you still here?

**Comment by pernielsentikaer** [comment]
All good, it might be a simple thing, but wanted to make sure nothing was missing

**Comment by pernielsentikaer** [comment]
Is this ready?

**Comment by bkeys818** [comment]
@pernielsentikaer, Yup!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**